apps/server/src/modules/federal-state/controller/mapper/federal-state.mapper.ts

```diff
@@ -1,16 +1,29 @@
-import { FederalStateDO } from '../../domainobject/federal-state.do';
+import { FederalStateDO, ICounty } from '../../domainobject/federal-state.do';
+import { CountyResponse } from '../dto';
 import { FederalStateResponse } from '../dto/federal-state.response';
 
 export class FederalStateMapper {
-	static mapToResponse(federalStateDO: FederalStateDO): FederalStateResponse {
+	static mapFederalStateToResponse(federalStateDO: FederalStateDO): FederalStateResponse {
 		const dto = new FederalStateResponse({
-			id: federalStateDO.id,
+			_id: federalStateDO.id,
 			name: federalStateDO.name,
 			abbreviation: federalStateDO.abbreviation,
+			counties: federalStateDO.counties
+				? federalStateDO.counties.map((county) => this.mapCountyToResponse(county))
+				: undefined,
 			logoUrl: federalStateDO.logoUrl,
 			createdAt: federalStateDO.createdAt,
 			updatedAt: federalStateDO.updatedAt,
 		});
 		return dto;
 	}
+
+	static mapCountyToResponse(county: ICounty) {
+		const countyResponse = {
```

Comment:
can add a county response class, and here instantiate that class instead of a simple object with no type